crates/agent/src/controllers/collection.rs

```diff
@@ -75,7 +75,11 @@ impl CollectionStatus {
         } else {
             None
         };
-        let periodic_next = periodic::next_periodic_publish(state);
+        let periodic_next = if model.derive.is_some() {
```

Comment:
Should this not also check whether the derivation is active, like you do down [here](https://github.com/estuary/flow/pull/1830/files#diff-941bb1e08a9eeb14ecb2d4063a56b2da43671068a6d0681794979aa6403e9b03R80)? Or is that not applicable here?

Reply:
Now that you mention it, I think I can just remove this whole conditional. `next_periodic_publish` already returns `None` if `!is_enabled_task(state)`